packages/tools/generator-component/generators/app/templates/src/components/Skeleton.vue

```diff
@@ -1,36 +1,36 @@
 <template>
     <div
-        :data-theme="theme"
         :class="$style['c-<%= name.class %>']"
-        data-test-id='<%= name.class %>-component'>
+        data-test-id="<%= name.class %>">
         {{ copy.text }}
```

Comment:
copy does not exist if you have not selected i18n

Reply:
Good spot – will update

Reply:
Fixed